packages/shared-ux/button_toolbar/src/buttons/toolbar_button/toolbar_button.tsx

```diff
@@ -130,7 +130,7 @@ const ToolbarStandardButton = ({
     <EuiButton
       size={rest.size}
       isDisabled={isDisabled}
-      css={cssProps}
+      css={type === 'primary' ? null : cssProps}
```

Comment:
Adopting this removes the style definitions for grouped buttons when the primary type is passed. We could instead do the following to yield the same result without modifying this prop;

```diff
diff --git a/packages/shared-ux/button_toolbar/src/buttons/toolbar_button/toolbar_button.tsx b/packages/shared-ux/button_toolbar/src/buttons/toolbar_button/toolbar_button.tsx
index a120c3184ff..4ec5da00b7e 100644
--- a/packages/shared-ux/button_toolbar/src/buttons/toolbar_button/toolbar_button.tsx
+++ b/packages/shared-ux/button_toolbar/src/buttons/toolbar_button/toolbar_button.tsx
@@ -88,7 +88,7 @@ const computeToolbarButtonCommonCSSProps = (
       : {};
 
   const defaultStyles = {
-    ...toolButtonStyles.default,
+    ...(type === 'primary' ? {} : toolButtonStyles.default),
     ...groupPositionStyles,
   };
 
@@ -96,7 +96,7 @@ const computeToolbarButtonCommonCSSProps = (
     ? defaultStyles
     : {
         ...defaultStyles,
-        ...(type === 'primary' ? {} : toolButtonStyles.emptyButton),
+      ...(type === 'empty' ? toolButtonStyles.emptyButton : {}),
       };
 };
 ```

Reply:
Thanks for the review and suggestion @eokoneyo !
Changes pushed